share/ruby-install/jruby/functions.sh

```diff
@@ -11,7 +11,7 @@ ruby_url="${ruby_url:-$ruby_mirror/$ruby_version/$ruby_archive}"
 function install_ruby()
 {
 	log "Installing jruby $ruby_version ..."
-	cp -R "$src_dir/$ruby_dir_name" "$install_dir" || return $?
+	cp -R "$src_dir/$ruby_dir_name" "$rubies_dir/" || return $?
```

Comment:
One minor issue here. Users can override `install_dir` via `--install-dir`.

Reply:
Copying the contents of `$ruby_dir_name` to `$install_dir` seems like a nice solution. @donv ^

Reply:
Seems good to me.